ResearchKit/ActiveTasks/ORKTappingContentView.m

```diff
@@ -116,7 +116,10 @@ - (void)tintColorDidChange {
 }
 
 - (void)setTapCount:(NSUInteger)tapCount {
-    _tapCountLabel.text = [NSString stringWithFormat:@"%02lu", (unsigned long)tapCount];
+    NSNumberFormatter *formatter = [NSNumberFormatter new];
+    formatter.locale = [NSLocale currentLocale];
+    formatter.minimumIntegerDigits = 2;
+    _tapCountLabel.text = [NSString stringWithFormat:@"%2@", [formatter stringFromNumber:@(tapCount)]];
```

Comment:
The tapCount is being updated very frequently, better to retain this formatter with `ORKLocalizedStringFromNumber`(create once).

Also `stringWithFormat:` is redundant.